oauth-keycloak-authorizer/src/main/java/io/strimzi/kafka/oauth/server/authorizer/KeycloakRBACAuthorizer.java

```diff
@@ -160,258 +156,123 @@
  * This authorizer honors the <em>super.users</em> configuration. Super users are automatically granted any authorization request.
  * </p>
  */
-@SuppressWarnings("deprecation")
-public class KeycloakRBACAuthorizer extends AclAuthorizer {
-
-    private static final String PRINCIPAL_BUILDER_CLASS = OAuthKafkaPrincipalBuilder.class.getName();
-    private static final String DEPRECATED_PRINCIPAL_BUILDER_CLASS = JwtKafkaPrincipalBuilder.class.getName();
+public class KeycloakRBACAuthorizer implements Authorizer {
```

Comment:
I guess we can't deprecate this class because the `KeycloakAuthorizer` can delegate to it. But how are we expecting a user to know they need to move to `KeycloakAuthorizer`?